Containerfile

```diff
@@ -25,3 +25,17 @@ RUN set -e && for OCP_VERSION in $(ls -1 ocp-product-docs-plaintext); do \
 FROM scratch
 COPY --from=lightspeed-rag-builder /workdir/vector_db/ocp_product_docs /rag/vector_db/ocp_product_docs
 COPY --from=lightspeed-rag-builder /workdir/embeddings_model /rag/embeddings_model
+
+# Labels for enterprise contract
+LABEL com.redhat.component=openshift-lightspeed
+LABEL description="Red Hat OpenShift Lightspeed - AI assitant for managing OpenShift clusters."
+LABEL distribution-scope=public
```

Comment:
Do we want this public , we dont release Rag content right ?

Reply:
by that logic this image shouldn't be checked against EC because we don't ship it.

Reply:
Found possible values of distribution-scope in https://github.com/projectatomic/ContainerApplicationGenericLabels, made it private.